st3/lsp_utils/server_npm_resource.py

```diff
@@ -78,10 +77,10 @@ def _copy_to_cache(self):
         if os.path.isdir(cache_server_path):
             # Server already in cache. Check if version has changed and if so, delete existing copy in cache.
             try:
-                src_package_json = json.loads(ResourcePath(src_path, 'package.json').read_text())
-                dst_package_json = json.loads(ResourcePath(dst_path, 'package.json').read_text())
+                src_package_json = ResourcePath(src_path, 'package.json').read_text()
+                dst_package_json = ResourcePath(dst_path, 'package.json').read_text()
 
-                if src_package_json['version'] != dst_package_json['version']:
+                if src_package_json != dst_package_json:
```

Comment:
I don't think we had a reason to worry about the performance of the previous solution but with your change, we'll pick up any change to the file so that seems like a positive thing.